File: CursoJava/Exer/src/Outros/ListaAlgoritimo/Algoritimo3/Exer07.java
/*
7) Escrever um algoritmo que leia um valor para uma variável N de 1 a 10 e calcule a tabuada de
N. Mostre a tabuada na forma: 0 x N = 0, 1 x N = 1N, 2 x N = 2N, ..., 10 x N = 10N.
 */

package Outros.ListaAlgoritimo.Algoritimo3;

public class Exer07 {
    public static void main(String[] args) {
        int n = 5;
        for (int i = 0; i <= 10; i++) {
            System.out.println(i + " x " + n + " = " + (i * n));
        }
    }
}

File: CursoLuizOtavio/CursoPython3/exer/lista_algoritimos/algoritimo3/exer07.py
"""
7) Escrever um algoritmo que leia um valor para uma variável N de 1 a 10 e calcule a tabuada de
N. Mostre a tabuada na forma: 0 x N = 0, 1 x N = 1N, 2 x N = 2N, ..., 10 x N = 10N.
"""

num = int(input("Digite um número de 1 a 10: "))
for i in range(0, 11):
    print(f"{i} x {num} = {i * num}")
